src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/regelsett/lov2021/standard/TomForskyver.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.lov2021.standard
+
+import no.nav.familie.ks.sak.common.util.DATO_LOVENDRING_2024
+import no.nav.familie.ks.sak.common.util.sisteDagIMåned
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.Vilkår
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.domene.VilkårResultat
+import no.nav.familie.ks.sak.kjerne.behandling.steg.vilkårsvurdering.regelsett.TilknyttetVilkårResultater
+import java.time.LocalDate
+
+private val sisteMuligeTomForBarnetsAlderILov2021 = DATO_LOVENDRING_2024.minusDays(1)
+
+fun forskyvTom(
+    tilknyttetVilkårResultater: TilknyttetVilkårResultater,
+): LocalDate? {
+    val periodeTom = tilknyttetVilkårResultater.gjeldende.periodeTom
+    if (periodeTom == null) {
+        return null
+    }
```

Comment:
```suggestion
    val periodeTom = tilknyttetVilkårResultater.gjeldende.periodeTom ?: return null
```

Smak og behag, men intellij slenger ut dette ihvertfall haha.

Reply:
Syns det er litt mer lesbar kode om slike "program flow" statements er litt mer i ansiktet mitt, men ja, veldig smak og behag det der 😅 Beholder det slik som det er inntil videre, men bra forslag uansett!